docs/user-manual/engine/standalone.md

```diff
@@ -0,0 +1,282 @@
+
+    // create a box
+    const box = new pc.Entity();
+    box.addComponent('model', {
+        type: 'box'
+    });
+    app.root.addChild(box);
+
+    // rotate the box
+    app.on('update', (dt: number) => box.rotate(10 * dt, 20 * dt, 30 * dt));
+    ```
+
+5. Now, run the Vite development server:
+
+    ```sh
+    npx vite
+    ```
+
+    This starts a server at `http://localhost:5173`. Visit this URL in your browser and you will see the following:
+
+    ![Vite Cube](/images/user-manual/engine/standalone/vite-cube.webp)
+
+    :::tip
+
+    Every time you save your source files, the tab will reload automatically. 
+
+    :::
+
+  </TabItem>
+</Tabs>
+
+## Option 2: Import Map and CDN
+
+An [import map](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script/type/importmap) can resolve module specifiers in JavaScript modules. Consider this import statement:
+
+```javascript
+import * as pc from 'playcanvas';
+```
+
+An import map can resolve `playcanvas` to a CDN-hosted build of the engine that can be dynamically loaded by the browser. This means that we can skip the build step described in Option 1.
+
+1. Open a Terminal/Command Prompt, create a folder for your app and `cd` inside it.
+2. Create an `index.html` and paste this:
+
+    ```html title="index.html"
+    <!DOCTYPE html>
+    <html lang="en">
+        <head>
+            <meta charset="utf-8">
+            <style>
+                body { margin: 0; overflow: hidden; }
+            </style>
+            <script type="importmap">
+            {
+                "imports": {
+                    "playcanvas": "https://cdn.skypack.dev/playcanvas"
+                }
+            }
+            </script>
+        </head>
+        <body>
+            <script type="module" src="main.js"></script>
+            <canvas id='application'></canvas>
+        </body>
+    </html>
+    ```
+
+3. Create a `main.js` and paste this:
+
+    ```javascript title="main.js"
+    import * as pc from 'playcanvas';
+
+    // create an application
+    const canvas = document.getElementById('application');
+    const app = new pc.Application(canvas);
+    app.setCanvasResolution(pc.RESOLUTION_AUTO);
+    app.setCanvasFillMode(pc.FILLMODE_FILL_WINDOW);
+    app.start();
+
+    // create a camera
+    const camera = new pc.Entity();
+    camera.addComponent('camera', {
+        clearColor: new pc.Color(0.3, 0.3, 0.7)
+    });
+    camera.setPosition(0, 0, 3);
+    app.root.addChild(camera);
+
+    // create a light
+    const light = new pc.Entity();
+    light.addComponent('light');
+    light.setEulerAngles(45, 45, 0);
+    app.root.addChild(light);
+
+    // create a box
+    const box = new pc.Entity();
+    box.addComponent('model', {
+        type: 'box'
+    });
+    app.root.addChild(box);
+
+    // rotate the box
+    app.on('update', (dt) => box.rotate(10 * dt, 20 * dt, 30 * dt));
+    ```
+
+5. Now, install and run `serve`:
+
+    ```sh
+    npm install serve --save-dev
```

Comment:
You don't need to manually install `npm install serve` as `npx serve` will installs and run

Reply:
Thanks, updated.